website/docs/installation/windows-install/index.md

```diff
@@ -1,28 +1,124 @@
+## Installing Podman Desktop
 
-- [Installing silently with the Windows installer](/docs/installation/windows-install/installing-podman-desktop-silently-with-the-windows-installer)
-- [Installing with Chocolatey](/docs/installation/windows-install/installing-podman-desktop-with-chocolatey)
-- [Installing with Scoop](/docs/installation/windows-install/installing-podman-desktop-with-scoop)
-- [Installing with Winget](/docs/installation/windows-install/installing-podman-desktop-with-winget)
-- [Installing Podman Desktop and Podman in a restricted environment](/docs/installation/windows-install/installing-podman-desktop-and-podman-in-a-restricted-environment)
+To install Podman Desktop:
 
-#### Procedure
+1. [Download the Windows installer](/downloads/windows).
+
+1. To start the Podman Desktop installer, open the downloaded file.
+
+   ![Podman Desktop Setup installing](img/podman-desktop-setup-installing.png)
+
+<details>
+<summary>
+Alternatively, when your environment requires another installation method, consider installing with:
+- Silent Windows installer
+- Chocolatey
+- Scoop
+- Winget
+</summary>
+
+#### Silent Windows installer
 
 1. [Download the Windows installer](/downloads/windows).
 
-2. Open the downloaded file to start the Podman Desktop installer.
+1. To install the Podman Desktop Windows installer without user interaction, run the Windows installer with the silent flag `/S` from the Command Prompt:
+
+   ```shell-session
+   > podman-desktop-1.6.4-setup-x64.exe /S
+   ```
+
+#### Chocolatey
+
+1. Install the [Chocolatey package manager](https://chocolatey.org/install).
+
+1. To install the [Podman Desktop Chocolatey package](https://community.chocolatey.org/packages/podman-desktop), run the command from the Command Prompt:
+
+   ```shell-session
+   > choco install podman-desktop
+   ```
+
+#### Scoop package manager for Windows
+
+1. [Install the Scoop package manager](https://github.com/ScoopInstaller/Install#readme).
+
+1. To install the [Podman Desktop Scoop package](https://scoop.sh/#/apps?q=podman-desktop&s=0&d=1&o=true), run the commands from the Command Prompt:
+
+   ```shell-session
+   > scoop bucket add extras
+   > scoop install podman-desktop
+   ```
+
+#### Winget
+
+1. [Install the Winget Package manager for Windows](https://aka.ms/getwinget).
+
+1. To install the [Podman Desktop Winget package](https://winget.run/pkg/RedHat/Podman-Desktop), run the command from the Command Prompt:
+
+   ```shell-session
+   > winget install -e --id RedHat.Podman-Desktop
+   ```
+
+</details>
+
+## Installing Podman
+
+On Windows, running the Podman container engine requires running a Linux distribution on a virtual machine.
+
+Podman Desktop can assist you to install the Podman container engine in a Fedora distribution of Linux, on a [Windows Subsystem for Linux version 2 (WSL 2)](https://learn.microsoft.com/en-us/windows/wsl/about#what-is-wsl-2) virtual machine: the Podman Machine.
```

Comment:
I don't think we need to mention WSL again? Bit of a confusing part.

We could summarize it as "Podman Desktop creates a virtual machine using `podman machine` that runs Fedora on WSL2" and just hyperlink WSL2 to the "what is wsl2" on microsoft's site.

Reply:
```suggestion
Podman Desktop creates a [Windows Subsystem for Linux version 2 (WSL 2)](https://learn.microsoft.com/en-us/windows/wsl/about#what-is-wsl-2) virtual machine: the Podman Machine.
```